src/pages/library/Showcase.js

```diff
@@ -44,10 +44,10 @@ const makeCard = variant => ({ workspace: { namespace, name, attributes: { descr
       style: {
         backgroundRepeat: 'no-repeat', backgroundPosition: 'center', backgroundSize: 'auto 100%', borderRadius: '0 5px 5px 0',
         width: 87,
-        ...Utils.switchCase(variant,
-          ['new', () => ({ backgroundImage: `url(${covidBg})` })],
-          ['gatk', () => ({ backgroundColor: '#333', backgroundImage: `url(${gatkLogo})`, backgroundSize: undefined })],
-          [Utils.DEFAULT, () => ({ backgroundImage: `url(${featuredBg})`, opacity: 0.75 })]
+        ...Utils.cond(
+          [name.toLowerCase().includes('covid'), { backgroundImage: `url(${covidBg})` }],
+          [variant === 'gatk', () => ({ backgroundColor: '#333', backgroundImage: `url(${gatkLogo})`, backgroundSize: undefined })],
```

Comment:
If a workspace card is both a `gatk` variant and contains `covid` in the name, which image should we use? As is, this will favor the COVID background. If that's not the desire, these lines will need to be swapped.

Reply:
Ahh that's a good point. Is there a person I should talk to in order to clarify which to prioritize?

Reply:
I would start with Amelia or Adrian. If they can't answer the question, I bet they could tell you who to ask (probably someone in Customer Success).